pyrobosim/pyrobosim/data/example_location_data_accessories.yaml

```diff
@@ -35,19 +35,19 @@ charger:
       nav_poses:
         - position:  # below
             x: 0.0
-            y: -0.5
+            y: -0.35
           rotation_eul:
             yaw: 1.57
         - position:  # left
-            x: -0.35
+            x: -0.5
             y: 0.0
         - position:  # above
             x: 0.0
-            y: 0.5
+            y: 0.35
           rotation_eul:
             yaw: -1.57
         - position:  # right
-            x: 0.35
+            x: 0.5
             y: 0.0
           rotation_eul:
             yaw: 3.14
```

Comment:
Incidental fix -- the charger's X and Y nav poses were wrong, so this wasn't working